src/Microsoft.AspNetCore.BuildTools.ApiCheck/build/Microsoft.AspNetCore.BuildTools.ApiCheck.targets

```diff
@@ -0,0 +1,30 @@
+﻿<!--
```

Comment:
This breaks one of our design patterns for tools and targets. To keep our tools composible and to separate concerns, tools packages only deliver the tool. Integration into build cycle is specified by KoreBuild and/or Internal.AspNetCore.Sdk. 

For example, the NuGetPackageVerifier package contains the tool, but the target to light it up actually is [defined in KoreBuild](https://github.com/aspnet/KoreBuild/blob/f692ca7c8ef8a277b92aef6cfc85ffdda2beafc8/build/targets/KoreBuild.SolutionBuild.targets#L344-L346).
Likewise, the `GetGitCommitInfo` task is defined and delivered by Internal.AspNetCore.BuildTools.Tasks, but the target that ties it into MSBuild compile step is [defined in Internal.AspNetCore.Sdk](https://github.com/aspnet/BuildTools/blob/e78bac3125d8baa0791205e22b884efa9224ba2f/src/Internal.AspNetCore.Sdk/build/Git.targets).

This fix should be simple. Lift the ApiCheck target into Internal.AspNetCore.Sdk instead.